Jenkinsfile

```diff
@@ -171,6 +171,19 @@ podTemplate(cloud: 'openshift', label: 'coreos-assembler', yaml: pod, defaultCon
             }
         }
 
+        stage('QEMU Kola Run') {
+            utils.shwrap("""
+            coreos-assembler kola run
+            tar -cf - tmp/kola/ | xz -c9 > _kola_temp.tar.xz
```

Comment:
Going with `|| :` for this PR.

As far as the `--exit-zero-on-test-failures` it can be hard to make distinctions about what failures are environment (AN: cloud) related and what are legitimate test failures from the runner's perspective. I'll see if I can spend some time thinking up some potential ways that could potentially do something like that and write up a mantle issue if I come up with anything.

Reply:
> it can be hard to make distinctions about what failures are environment (AN: cloud) related and what are legitimate test failures from the runner's perspective

Yeah, agreed (we've had to deal with this with PAPR too). Though for the qemu target at least, we should be able to rule those out. :)